code/game/gamemodes/events/holidays/Holidays.dm

```diff
@@ -93,6 +93,14 @@ var/global/list/Holiday = list() //Holidays are lists now, so we can have more t
 			switch(DD)
 				if(1)
 					Holiday["Interstellar Workers' Day"] = "This holiday celebrates the work of laborers and the working class."
+				
+				if(18)
+					Holiday["Remembrance Day"] = "Rembrance Day (or, as it is more informally known, Armistice Day) is confederate-wide holiday mostly observed \ 

```

Comment:
Remembrance Day [...] is a confederation-wide holiday mostly observed in its member states [...] Officially [...] Unofficially [...] Human-Unathi War [...] and the wearing of poppies